.github/workflows/reusable-jira-workflow.yml

```diff
@@ -0,0 +1,36 @@
+name: Jira
+
+on:
+  workflow_call:
+    inputs:
+      branch_name:
+        required: true
+        type: string
+
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Check Jira Story
+      shell: pwsh
+      run: |
+            Install-Module JiraPS -Scope CurrentUser -Force
+            if ("${{ inputs.branch_name }}" -like "*renovate/*" -eq $True)
+            {
+              Write-Host "Skipping, renovate branch detected"
+              return
+            }
+
+            Set-JiraConfigServer -Server "${{ secrets.JIRA_URL }}"
+            if("${{ inputs.branch_name }}" -match "[A-za-z]+[_-]+[0-9]+" -eq $False)
```

Comment:
Should it be `^feature/[a-zA-Z]+[_-][0-9]+` or `/[a-zA-Z]+[_-][0-9]+`?

Reply:
good suggestion, we'll discuss with the team as a follow up on this